presets/4.3/rates/Tehllama_Nytfury_533_Limon_RATEBANK.txt

```diff
@@ -0,0 +1,118 @@
+#$ TITLE: Racing RateBank: Nytfury, HeadsUp-533, Limon, Tehllama 
+#$ FIRMWARE_VERSION: 3.5
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: Race, Rates, Racing, Bank, Nytfury, HeadsUp, 533, Limon, Beginner
+#$ AUTHOR: Daniel Appel / Tehllama
+#$ DESCRIPTION: These are some proven race rates for pilots to use
+#$ DESCRIPTION: Use selected options to change any RateProfile values (will retain RateProfile 1 by default)
```

Comment:
worth mentioning what this preset is doing.
Which rates go to which profile
and what options are doing.